test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -1139,7 +1139,7 @@ public void ProcessResults_ValueTypePropertyWithBindRequired_RequiredValidatorIg
                 .Value;
             var error = Assert.Single(entry.Errors);
             Assert.Null(error.Exception);
-            Assert.Equal("A value for the 'ValueTypeProperty' property was not provided.", error.ErrorMessage);
+            Assert.Equal("A value for the 'theModel.ValueTypeProperty' property was not provided.", error.ErrorMessage);
```

Comment:
Yeah let's talk tomorrow since I don't understand what you mean. Note this change to `MutableObjectModelBinder` makes it more consistent with for example the `ErrorMessage` that `TryAddModelError()` adds in the `FormatException` special case.